modules/components/pages/inputs/spicedb_watch.adoc

```diff
@@ -0,0 +1,260 @@
+= spicedb_watch
+// tag::single-source[]
+:type: input
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+Consumes messages from the https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WatchService.Watch[Watch API^] of a https://authzed.com/docs/spicedb/getting-started/discovering-spicedb[SpiceDB^] instance. This input is useful if you have downstream applications that need to react to real-time changes in data managed by SpiceDB.
+
+ifndef::env-cloud[]
+Introduced in version 4.39.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common configuration fields, showing default values
+input:
+  label: ""
+  spicedb_watch:
+    endpoint: grpc.authzed.com:443 # No default (required)
+    bearer_token: ""
+    cache: "" # No default (required)
+```
+--
+Advanced::
++
+--
+```yml
+# All configuration fields, showing default values
+input:
+  label: ""
+  spicedb_watch:
+    endpoint: grpc.authzed.com:443 # No default (required)
+    bearer_token: ""
+    max_receive_message_bytes: 4MB
+    cache: "" # No default (required)
+    cache_key: authzed.com/spicedb/watch/last_zed_token
+    tls:
+      enabled: false
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+```
+--
+======
+
+== Authentication
+
+For this input to authenticate with your SpiceDB instance, you must provide:
+
+- The endpoint of the SpiceDB instance
+- A Bearer token
+
+== Configure a Cache
+
+You must use a xref:configuration:resources.adoc[cache resource] to store the https://authzed.com/docs/spicedb/concepts/consistency#zedtokens[ZedToken^] (ID) of the latest message consumed and acknowledged by this input. Ideally, the cache should persistent across restarts. This means that every time the input is initialized, it starts reading from the newest data updates.
+
+== Fields
+
+=== `endpoint`
+
+The endpoint of your SpiceDB instance.
+
+*Type*: `string`
+
+
+```yml
+# Examples
+endpoint: grpc.authzed.com:443
+```
+
+=== `bearer_token`
+
+The SpiceDB Bearer token to use to authenticate with your SpiceDB instance.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `max_receive_message_bytes`
+
+The maximum message size (in bytes) this input can receive.
+
+*Type*: `string`
+
+*Default*: `4MB`
+
+```yml
+# Examples
+
+max_receive_message_bytes: 100MB
+
+max_receive_message_bytes: 50mib
+```
+
+=== `cache`
+
+The xref:configuration:resources.adoc[cache resource] to use for performing unread message backfills. The ZedToken (ID) of the last message received must be stored in this cache, which is then used for subsequent requests.
```

Comment:
This is not accurate (my fault as I wrote this).
It should be `The cache resource to use to store the ZedToken of the last message processed. The last ZedToken is stored in the cache in the ACK function of the message which means that's only going to be stored when the message successfully went through all processors and outputs.`

Reply:
Reworded. Please check for accuracy.